### PR TITLE
DCOS-12405: Conditionally show declined offers funnel &table

### PR DESCRIPTION
**Pull request by jfurrow**
This PR hides the declined offers funnel & table data from a service's debug tab until the service has been waiting or over 5 minutes.

### CODE REVIEW

**Comment by jfurrow** [comment]
retest this please

File: plugins/services/src/js/containers/pod-debug/PodDebugContainer.js [hunk 1]
```diff
@@ -145,7 +147,8 @@ class PodDebugTabView extends React.Component {
     let mainContent = null;
     let offerCount = null;
 
-    if (queue == null || queue.declinedOffers.summary == null) {
+    if (!DeclinedOffersUtil.shouldDisplayDeclinedOffersWarning(queue)
+      || queue.declinedOffers.summary == null) {
```
Comment (mlunoe) [inline plugins/services/src/js/containers/pod-debug/PodDebugContainer.js]: Couldn't we default assign these values and let go of the `if`-`else` here?
Not part of this PR, but would be nice with a little cleanup here.

File: plugins/services/src/js/containers/service-debug/ServiceDebugContainer.js [hunk 2]
```diff
@@ -117,7 +118,8 @@ class ServiceDebugContainer extends React.Component {
       } else {
         introText = 'Rejected offer analysis is not currently supported.';
       }
-    } else if (queue == null || queue.declinedOffers.summary == null) {
+    } else if (!DeclinedOffersUtil.shouldDisplayDeclinedOffersWarning(queue)
```
Comment (mlunoe) [inline plugins/services/src/js/containers/service-debug/ServiceDebugContainer.js]: #nit We should clean these `if`-`else` and nested `if`s up at some point too. (Again not part of this PR)

**Comment by mlunoe** [comment]
Run integration tests

File: plugins/services/src/js/containers/pod-debug/PodDebugContainer.js (hunk 1 above)
Comment (jfurrow) [inline plugins/services/src/js/containers/pod-debug/PodDebugContainer.js]: Yeah, we can definitely clean these up! I created a JIRA: https://mesosphere.atlassian.net/browse/DCOS-13205

File: plugins/services/src/js/containers/service-debug/ServiceDebugContainer.js (hunk 2 above)
Comment (jfurrow) [inline plugins/services/src/js/containers/service-debug/ServiceDebugContainer.js]: JIA: https://mesosphere.atlassian.net/browse/DCOS-13205

**Comment by jfurrow** [comment]
Rebased with integration test fixes 🎉

**Comment by rcorral** [comment]
Works well :)
